### DISCUSSION TITLE
How to perform backup of meta-data directories of Doris FE, can we do it when the Doris is running?

### DISCUSSION THREAD

**Discussion by patrickdung2022** [apache/doris#31939] [A - General / Q&A]
For web pages like https://doris.apache.org/docs/admin-manual/cluster-management/upgrade/
it mentioned

```
metadata backup (important)

Make a full backup of the doris-meta directory of the FE-Master node! 
```
But it does not provide information like whether Doris FE master node should be stopped.

Understand that the backup of doris-meta is for upgrading.

Besides upgrading, how about production use cases?
How to make a (consistent) backup the doris-meta of a three node FE cluster?
Do we need to shutdown the FE node?
As there any SQL/system command that can generate a consistent meta data backup?

Thank you in advance.

**Comment by yagagagaga** [comment]
The Doris FE master node should be stopped when metadata backing up. You can use [manager](https://doris.apache.org/ecosystem/cluster-management) to do upgrade.

**Reply by SparkElf** [reply]
the document said that doris metadata supports hot backup?
